deltachat-ios/Controller/AccountSetup/SecuritySettingsController.swift

```diff
@@ -28,15 +20,8 @@ class SecuritySettingsController: UITableViewController {
         }
     }
 
-    init(dcContext: DcContext, title: String, type: SecurityType) {
-        self.securityType = type
-        self.dcContext = dcContext
-        switch securityType {
-        case .IMAPSecurity:
-            selectedIndex = options.firstIndex(of: Int32(dcContext.getConfigInt("mail_security"))) ?? 0
-        case .SMTPSecurity:
-            selectedIndex = options.firstIndex(of: Int32(dcContext.getConfigInt("send_security"))) ?? 0
-        }
+    init(initValue: Int, title: String) {
+        selectedIndex = options.firstIndex(of: Int32(initValue)) ?? 0
```

Comment:
Setting `0` as default feels kind of weird, but I suspect it being another "this PR does not clean up"?

Reply:
that :)

but it also makes sense: when the database holds some weird value that does not map to an existing option (maybe it was sth. removed - or sth. comes from a backup from eg. android with a newer corer) - we select just the first item - which is the default resp. "Automatic"